test/new-e2e/runner/profile.go

```diff
@@ -17,10 +17,11 @@ import (
 type CloudProvider string
 
 const (
-	AWS       CloudProvider = "aws"
-	Azure     CloudProvider = "az"
-	GCP       CloudProvider = "gcp"
-	EnvPrefix               = "E2E_"
+	AWS                   CloudProvider = "aws"
+	Azure                 CloudProvider = "az"
+	GCP                   CloudProvider = "gcp"
+	EnvPrefix                           = "E2E_"
+	DefaultCISecretPrefix               = "ci.datadog-agent."
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
As this is CI related, would define it in `ci-profile.go` instead

Reply:
Same for `defaultCIEnvs`